### PR TITLE
alphabetizing and adding d2l-localize-behavior

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

**Comment by dlockhart** [comment]
Figured we should have these in alphabetical order to make finding stuff easier and to limit the chance of a conflict if people keep adding to the end of the list.

File: dependency-map.txt [hunk 1]
```diff
@@ -1,8 +1,9 @@
+d2l-button,d2l-button,git+https://github.com/BrightspaceUI/button.git#polymer-3
 d2l-colors,d2l-colors,BrightspaceUI/colors#semver:^4.0.0
-d2l-typography,d2l-typography,BrightspaceUI/typography#semver:^7.0.0
+d2l-icons,d2l-icons,git+https://github.com/BrightspaceUI/icons.git#polymer-3
+d2l-fastdom-import,fastdom,^1.0.8
+d2l-link,d2l-link,git+https://github.com/BrightspaceUI/link.git#polymer-3
+d2l-localize-behavior,BrightspaceUI/localize-behavior#semver:^2.0.0
```
Comment (dbatiste) [inline dependency-map.txt]: @svanherk : Curious about the name being listed twice, followed by the reference.  I noticed Dave's localize behavior here only has the name once.

Comment (dbatiste) [reply dependency-map.txt]: Also notice that fastdom is like this `d2l-fastdom-import,fastdom,^1.0.8`.

Comment (dlockhart) [reply dependency-map.txt]: Oh whoops, that could have totally been my bad.

Comment (dlockhart) [reply dependency-map.txt]: I'm guessing it lets us re-map a dependency to another, like what Anthony did with fastdom?

Comment (dlockhart) [reply dependency-map.txt]: Updated mine to list it out twice like the others.

Comment (svanherk) [reply dependency-map.txt]: Yeah I _believe_ this is how it works:
First value - dependency name in the old bower.json file that the modulizer needs to find a match for, acts as the key
Second value - The name of the npm dependency (what will be the key in `package.json`)
Third value - The value of that key in `package.json`

Comment (dbatiste) [reply dependency-map.txt]: Cool thanks.

**Pull request merged**